keps/sig-storage/2268-non-graceful-shutdown/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: non graceful shutdown
+kep-number: 2268
+authors:
+  - "@yastij"
+  - "@xing-yang"
+owning-sig: sig-storage
```

Comment:
Generally lgtm. I think since the volume attachment handling is a minor part of the proposal, we should swap sig-node and storage here.

Reply:
I think initially this KEP's owning sig is sig-node, but @yastij changed it to sig-storage at one point. The current KEP is a trimmed down version of his KEP when I took over. So I'd like to keep the owning sig as sig-storage and ask sig-node to also take a look.